auth/auth.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019, Square, Inc.
+
+// Package auth provides team-based authentication and authorization.
+package auth
+
+import (
+	"net/http"
+)
+
+type ACL struct {
+	// User-defined role. This must exactly match a Caller role for the ACL
+	// to match.
+	Role string
+
+	// Role grants admin access (all ops) to request. Mutually exclusive with Ops.
```

Comment:
"Ops" - is that a field that doesn't exist anymore?

Reply:
No, was copy-paste error. Fixed.